client/js/plugins/sails.js

```diff
@@ -115,6 +115,15 @@ io.socket.on('game', function (evData) {
         case 'reLogin':
           store.dispatch('updateGameThenResetPNumIfNull', evData.data.game);
           break;
+        case 'requestStalemate':
+          if (evData.data.requestedByPNum !== store.state.game.myPNum) {
+            store.commit('setConsideringOpponentStalemateRequest', true);
+          }
```

Comment:
I'd switch this to a fast fail for readability and not needing to nest.
```suggestion
          if (evData.data.requestedByPNum === store.state.game.myPNum) {
            return;
          }
          store.commit('setConsideringOpponentStalemateRequest', true);
```

Reply:
What's funky about this is that you still have the nested return. I think it's slightly more clear as-is